forge-gui/res/cardsfolder/upcoming/courageous_resolve.txt

```diff
@@ -3,7 +3,8 @@ ManaCost:2 W
 Types:Instant
 A:SP$ Pump | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select up to one target creature you control | TargetMin$ 0 | TargetMax$ 1 | SubAbility$ DBDraw | KW$ Protection:Card.ControlledBy Player.OpponentOf PlayerUID_ChosenPlayerUID,Emblem.ControlledBy Player.OpponentOf PlayerUID_ChosenPlayerUID:Protection from all opponents of ChosenPlayerName | DefinedKW$ You | SpellDescription$ Up to one target creature you control gains protection from each of your opponents until end of turn. Draw a card. (It can't be blocked, targeted, dealt damage, enchanted, or equipped by anything controlled by those players.)
```

Comment:
imo i would be more clean to use `DefinedKW$ Opponent` ?
then it would create a Protection keyword for each of your opponents

Reply:
Hmm don't know. using `DefinedKW$ Opponent` makes the card not work

Reply:
Of course you need to update the other part of the protection keyword too

Reply:
Hmm can't really figure out how to make this work. The KW$ syntax is kinda cryptic

Reply:
@tool4ever can you check this out?

i would prefer if it has a Keyword for each opponent

Reply:
Just this imo:
`KW$ Protection:Card.ControlledBy Player.PlayerUID_ChosenPlayerUID,Emblem.ControlledBy Player.PlayerUID_ChosenPlayerUID:Protection from ChosenPlayerName`